implementations/rust/ockam/ockam_entity/src/lib.rs

```diff
@@ -8,39 +8,102 @@
     unsafe_code,
     unused_import_braces,
     unused_qualifications,
-    // TODO re-enable warnings
+    // warnings
 )]
 
-use crate::EntityError;
+pub use change::*;
+pub use channel::*;
+pub use contact::*;
+pub use entity::*;
+pub use error::*;
+pub use identifiers::*;
+pub use key_attributes::*;
 use ockam_channel::SecureChannelVault;
-use ockam_core::lib::HashMap;
-use ockam_core::{Address, Result};
-use ockam_node::Context;
-use ockam_vault_core::{Hasher, KeyIdVault, SecretVault, Signer, Verifier};
-use ockam_vault_sync_core::VaultSync;
+use ockam_core::{lib::HashMap, Address, Message, Result};
+use ockam_node::{block_future, Context};
+use ockam_vault::{Hasher, KeyIdVault, SecretVault, Signer, Verifier};
+pub use profile_state::*;
+pub use traits::*;
+pub use worker::*;
+
+use crate::EntityError;
+
+pub struct Handle {
```

Comment:
Consider moving this to ockam_node (in separate PR maybe). It would be ideally to place it in ockam_core, but I believe it's not possible until we have Context/Node traits